core/jazz_services/config/global-config.json

```diff
@@ -97,6 +97,13 @@
             "editable": true,
             "isEmptyAllowed": false,
             "type": "Object"
+        },
+        {
+            "key": "deployment_targets",
+            "required": true,
```

Comment:
@svidhani If we are making this required, we'll need to add this to all the core services during bootstrapping of jazz stack. Currently, this happens in the installer. I'm creating an issue for us to make this change in the installer. Stack init might fail when we merge this change.

Reply:
@devsatishm are you suggesting to make this not required. anyways we have to backfill the data?